grammars/SeedPython.g4

```diff
@@ -13,75 +13,58 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * This is a derived work of https://github.com/bkiers/Python3-parser.
- * Here is the original license notice:
- *
- * The MIT License (MIT)
- *
- * Copyright (c) 2014 by Bart Kiers
- *
- * Permission is hereby granted, free of charge, to any person
- * obtaining a copy of this software and associated documentation
- * files (the "Software"), to deal in the Software without
- * restriction, including without limitation the rights to use,
- * copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the
- * Software is furnished to do so, subject to the following
- * conditions:
- *
- * The above copyright notice and this permission notice shall be
- * included in all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
- * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
- * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
- * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
- * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
- * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
- * OTHER DEALINGS IN THE SOFTWARE.
- *
- * Project      : python3-parser; an ANTLR4 grammar for Python 3
- *                https://github.com/bkiers/python3-parser
- * Developed by : Bart Kiers, bart@big-o.nl
- */
+ * SeedPyhton grammar is referred and modified from:
+ * https://docs.python.org/3.10/reference/grammar.html
+*/
 
 grammar SeedPython;
 
 import Common;
 
-tokens {
-  INDENT,
-  DEDENT
-}
+program: statements? EOF;
+
+statements: statement+;
+statement: compound_stmt | simple_stmts;
+simple_stmts:
+  simple_stmt (SEMICOLON simple_stmt)+ SEMICOLON? # multiple_simple_stmts
+  | simple_stmt                                   # single_simple_stmt;
+simple_stmt:
+  assignment         # assignment_placeholder
+  | star_expressions # expression_stmt
```

Comment:
Why do you define `expression_stmt` as `star_expressions`?

In https://github.com/bkiers/python3-parser/blob/master/src/main/antlr4/nl/bigo/pythonparser/Python3.g4 the name `star_expr` is defined as:

```
/// star_expr: ['*'] expr
star_expr
 : '*'? expr
 ;
```

But so far it's defined as below in our grammar:

```
star_expressions:
  star_expression (COMMA star_expression)+ COMMA? # mltiple_star_expressions
  | star_expression 
star_expression: expression;
```

I didn't find anything like `'*'? expr` in the grammar. It's not reasonable to still call it as "star_something".

Will our grammar become the same as the ANTLR's origin grammar a bit later? 

Since our grammar hasn't covered the real star-expr case like `a, *b = 1, 3`, or `def foo(a, *b)`, can we avoid using a confusing name for now?

Reply:
Agree that the star-expr will not be our requirement in very long time. So I remove all rules related to the star-expr to make it clear.